src/registrar/models/domain_application.py

```diff
@@ -493,6 +496,32 @@ def _send_confirmation_email(self):
         except EmailSendingError:
             logger.warning("Failed to send confirmation email", exc_info=True)
 
+    def _send_in_review_email(self):
+        """Send an email that this application is now in review.
+
+        The email goes to the email address that the submitter gave as their
+        contact information. If there is not submitter information, then do
+        nothing.
+        """
+        if self.submitter is None or self.submitter.email is None:
+            logger.warning(
+                "Cannot send status change (in review) email,"
+                "no submitter email address."
+            )
+            return
+        try:
+            logging.info(f"In review email sent to: {self.submitter.email}")
```

Comment:
**suggestion (non-blocking)**: As above, move the log statement after the send.

Reply:
Noted. What's the difference between `logger.info` and `logging.info`? I see `logger = logging.getLogger(__name__)` at the top of the doc but don't really get what it's doing.

And I assume those are app logs, so I should see them in my docker console and we can see them in cloud.gov's logs somewhere?